napari/_qt/qt_viewer.py

```diff
@@ -287,16 +287,19 @@ def _create_performance_dock_widget(self):
     def console(self):
         """QtConsole: iPython console terminal integrated into the napari GUI."""
         if self._console is None:
-            from .widgets.qt_console import QtConsole
+            try:
+                from napari_console import QtConsole
 
-            self.console = QtConsole({'viewer': self.viewer})
+                self.console = QtConsole(self.viewer)
+            except ImportError:
+                self._console = None
         return self._console
 
     @console.setter
     def console(self, console):
         self._console = console
-        self.dockConsole.setWidget(console)
-        self._update_theme()
```

Comment:
i assume the update theme stuff just isn't necessary anymore to keep styles in sync?

Reply:
ah so it's interesting, in the new package i connect to the viewer theme event internally, there are some pros/ cons around that so we might to think about if there is a cleaner way